durexforth.asm

```diff
@@ -97,13 +97,22 @@ _START = * + 1
 
 ; ----------- macros
 
-!set LINK = 0
-
-!macro BACKLINK {
-    ; it's tempting to add the string as a macro parameter,
-    ; but this does not seem to be supported by ACME.
-    !word	LINK
-    !set	LINK = * - 2
+!set TOP = $6fff
+!set DICTOP = TOP
+
+!set BACK = *
+* = DICTOP

```

Comment:
There was a comment earlier about naming. Since the dictionary stack grows downwards in memory, isn't it so that what is currently named TOP is actually the bottom? But I guess it depends if one view it as a stack or not. Naming is the hardest part!